app/lib/common/widgets/chat/convo_card.dart

```diff
@@ -134,12 +134,6 @@ class _ConvoCardState extends ConsumerState<ConvoCard> {
     await room.unmute();
     if (!mounted) return;
     EasyLoading.showToast(L10n.of(context).notificationsUnmuted);
-    await Future.delayed(const Duration(seconds: 1), () {
```

Comment:
Frankly I deleted this code, because I confirmed `maybeRoomProvider` is auto-updated without `ref.invalidate()`.
Of course, it was confirmed at another place.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1598/commits/96e14461f975f573fe0c69c947ba738bd9c70c10